test/api/logs.test.js

```diff
@@ -774,6 +774,49 @@ module.exports = () => {
         });
     });
 
+    describe('GET /api/logs/root', () => {
+        it('should return root logs only', (done) => {
+            request(server)
+                .get('/api/logs/root')
+                .expect(200)
+                .end((err, res) => {
+                    if (err) {
+                        done(err);
+                        return;
+                    }
+
+                    expect(res.body.data.every((log) => log.parentLogId === null));
+
+                    done();
+                });
+        });
+
+        it('should return an array', (done) => {
```

Comment:
You should put more explicit name, such as "should successfully return paginated logs" then based on seeders, you check different pagination (and you check the log ids)

Reply:
done